src/blocks/utils/SeamMeme/components/AppFeed.tsx

```diff
@@ -0,0 +1,14 @@
+import { AppFeedProps } from "../types/types";
+
+const AppFeed = ({ model, update }: AppFeedProps) => {
```

Comment:
For small components like this, prefer keeping them in a single file. This entire component can go inside of the FeedComponent, it doesn't need to be another separate component itself.